### PR TITLE
Fix #6358: Column sort icon disappears after selecting another column

### LINKED ISSUE #6358: DataTable: Column sort icon disappears after selecting another column
**Describe the defect**
Column sort icon gets removed after selecting another column. Regression from #3818. These new changes might be the cause https://github.com/primefaces/primefaces/commit/82abe07fb648d503e47487ac5fc715f1e3e8d325#diff-265f44d9cadc896ec3d21a3a721f15ddR2030-R2048

**Environment:**
 - PF Version: 9.0-SBAPSHOT
 - JSF + version: Mojarra 2.2.8 or MyFaces 2.3.6
 - Affected browsers: Chrome, iOS, ALL

**To Reproduce**
- Go http://localhost:8080/showcase/ui/data/datatable/columns.xhtml
- Sort ID asc
- Sort BRAND
- Icon of previous selected column disappear

**Expected behavior**
Sort icon should be reinitialized to unsort icon

@eduarddrenth Could you help us to sort this out?

### PR DESCRIPTION
**Pull request by dieuph**

### CODE REVIEW

**Comment by Rapster** [comment]
I think there might be an easier way to fix this:
image

**Comment by dieuph** [comment]
Hi @Rapster 
Thank you for point out this one. I fixed it following your way.

File: src/main/resources/META-INF/resources/primefaces/datatable/datatable.js [hunk 1]
```diff
@@ -2033,8 +2033,10 @@ PrimeFaces.widget.DataTable = PrimeFaces.widget.DeferredWidget.extend({
                                 $(PrimeFaces.escapeClientId(activeColumn.attr('id') + '_clone')).removeAttr('aria-sort').attr('aria-label', $this.getSortMessage(ariaLabelOfActive, $this.ascMessage));
                             }
 
-                            activeColumns.data('sortorder', $this.SORT_ORDER.UNSORTED).removeClass('ui-state-active')
-                                        .find('.ui-sortable-column-icon').removeClass('ui-icon-triangle-1-n ui-icon-triangle-1-s');
+                            activeColumns.data('sortorder', $this.SORT_ORDER.UNSORTED)
+                                         .removeClass('ui-state-active')
+                                         .find('.ui-sortable-column-icon')
+                                         .removeClass('ui-icon-triangle-1-n ui-icon-triangle-1-s');
```
Comment (Rapster) [inline src/main/resources/META-INF/resources/primefaces/datatable/datatable.js]: Please don't change the format specially if those lines are not part of the fix

**Comment by Rapster** [comment]
Thank you!

**Pull request merged**